### PR TITLE
fix(ui): checkbox and error message

### LINKED ISSUE #199: Fix/UI - SC CheckBox and Error Message
## The story:
As a developer, I want to have the checkbox and the error message changed to Styled Components.

---

## What we should do:
We should change the checkbox and error message to Styled Components.

---

## To be successful we need to:
_Check these off as they are completed:_
- [ ] Checkbox to SC
- [ ] Error Message to SC

---

### PR DESCRIPTION
**Pull request by suzynakayama**
## Fixed issues (PRs without this will not be accepted)
- Fixes #199 

## Changes
- Changed Checkbox to styled-components
- Changed Error Message to styled-components
- Changed Form Field to receive the props from the proper file
- Changes the Base CSS to delete the error message css

## PR Checklist
- [x]  Checkbox to SC
- [x]  Error Message to SC

### CODE REVIEW

File: packages/ui/components/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
-
 
 export const Checkbox: React.FunctionComponent<CheckboxProps> = ({
   indeterminate,
   defaultChecked,
   onChange,
-  text,
-  ...props
+  text
+  // ...props
```
Comment (quimgv) [inline packages/ui/components/Checkbox/Checkbox.tsx:17]: If we don't need this I'd remove it

File: packages/ui/components/Checkbox/Checkbox.tsx [hunk 2]
```diff
@@ -1,43 +1,43 @@
...
+  return (
+    <CheckboxDiv
+      className={classnames({
+        indeterminate,
+        checked
+      })}
+    >
+      <CheckboxInput
+        type="checkbox"
+        // {...props}
```
Comment (quimgv) [inline packages/ui/components/Checkbox/Checkbox.tsx:30]: Same here

File: packages/ui/components/Checkbox/Checkbox.styles.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+export interface InputProps extends CheckboxProps {
+  type?: string;
+  onChange?: (e: React.FormEvent<HTMLInputElement>) => void;
+  checked?: boolean;
+  defaultChecked?: boolean;
+}
+
+export interface SpanProps extends InputProps {}
```
Comment (quimgv) [inline packages/ui/components/Checkbox/Checkbox.styles.ts:16]: I've seen somewhere that if you don't need to add any specific prop you can just use InputProps to type whatever you need to type. @tristanMatthias do you know anything about this? What would you do?

**Review by quimgv — APPROVED** [review]
Great work @suzynakayama ! The code looks good to me aside from those small comments :)

File: packages/ui/components/Checkbox/Checkbox.styles.ts (hunk 3 above)
Comment (tristanMatthias) [reply packages/ui/components/Checkbox/Checkbox.styles.ts:16]: Either is fine :)

**Review by tristanMatthias — APPROVED** [review]
Thanks @suzynakayama :)

**Pull request merged**